### PR TITLE
Console should update active language when going through scrollback to match language of evaled code (fixes #2104)

### LINKED ISSUE #2104: console should update active language when going through scrollback to match language of evaled code
related to #2103 -- it seems like when you scroll back through the console with up/down, that the active language in the console should update to match the language/chunk type that was used in the original eval.

### PR DESCRIPTION
**Pull request by Hrtkk**
### Pull Request checklist

- [ ] **Documentation**: If this feature has or requires documentation, the relevant docs have been updated.
- [ ] **Changelog**: This PR updates the [changelog](https://github.com/iodide-project/iodide/blob/master/CHANGELOG.md) with any user-visible changes.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not

Console Menu syncing with the language type of history while scrolling back in the console.

### CODE REVIEW

File: .env~ [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+IODIDE_SERVER_DEBUG=True
+SECRET_KEY=59114b6a-2858-4caf-8878-482a24ee9542
+DJANGO_LOG_LEVEL=DEBUG
+DJANGO_LOG_FORMAT=json
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+GA_TRACKING_ID=
+EVAL_FRAME_ORIGIN=http://localhost:8000/
+EDITOR_ORIGIN=http://localhost:8000/
+APP_VERSION_STRING=dev
```
Comment (wlach) [inline .env~]: looks like this got accidentally added?

File: src/editor/console/input/reducer.js [hunk 2]
```diff
@@ -50,16 +51,21 @@ export default function reducer(state, action) {
       if (nextScrollback === 0) {
         // if we moved TO 0, set the consoleText from the cache
         nextConsoleText = consoleTextCache;
+        preConsoleLanguage =
+          historyLength > 0 ? inputHistory[historyLength - 1].language : "js";
```
Comment (wlach) [inline src/editor/console/input/reducer.js]:
```suggestion
          historyLength > 0 ? inputHistory[historyLength - 1].language : preConsoleLanguage;
```

Comment (Hrtkk) [reply src/editor/console/input/reducer.js]: Okay

**Comment by bcolloran** [comment]
closing in favor of #2508 (just moved over @Hrtkk's work from this PR, which was easier than rebasing)

thank you @Hrtkk!

**Pull request closed without merging**